05_timers/part01/test_time.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <time.h>
+
+#define SECS_IN_DAY (24 * 60 * 60)
+
+void show_time(clockid_t clock, const char *name)
+{
+    struct timespec ts, ts2;
+    clock_gettime(clock, &ts);
+    clock_getres(clock, &ts2);
+    long days = ts.tv_sec / SECS_IN_DAY;
+    printf("%s:\n\t\t %10ld.%ld\t resolution: %10jd.%09ld\t", name, ts.tv_sec, ts.tv_nsec, ts2.tv_sec, ts2.tv_nsec);
```

Comment:
Please split this line because it is too long. Please keep  MAX_LINE_SIZE = 80 columns.

Reply:
fixed line length and fix detail description in part01, part02